docs/cli.html

```diff
@@ -362,7 +362,7 @@ <h2 id="build">build</h2>
 <li>Generates and validates app definition.</li>
 <li>Detects dependencies via browserify (optional)</li>
 <li>Zips up all needed <code>.js</code> files. If you want to include more files, add a &quot;includeInBuild&quot; property (array with strings of regexp paths) to your <code>.zapierapprc</code>.</li>
-<li>Moves the zip to <code>build/build.zip</code></li>
+<li>Moves the zip to <code>build/build.zip</code> and <code>build/source.zip</code></li>
```

Comment:
Ah, this was my fault, I forgot to run `npm run docs` when merging that PR!

Reply:
I figured you had made some changes earlier! 👍